oemoflex/tools/labels.yaml

```diff
@@ -0,0 +1,12 @@
+!!python/tuple ['from', 'to', 'type', 'timeindex']: Timestamp
+!!python/tuple ['BB-biomass-st', 'BB-electricity', 'flow', 'Unnamed: 1_level_3']: Biomass
```

Comment:
```suggestion
('BB-biomass-st', 'BB-electricity', 'flow'): Biomass
```

Reply:
Would this work as well? I do not know this "python/tuple" approach, seems strange but maybe you know more?

Reply:
I researched a bit further on that, but I have not found another approach yet.
It is not recognized as a tuple if you are using ()
Instead, it is loaded as a string.
So ('A', 'B') in the yaml file becomes "('A', 'B')" in python.